packages/utilities/src/utils/collect-from-composed-schemas.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
```

Comment:
```suggestion
 * Copyright 2024 IBM Corporation.
```

(Not the only instance of this)